modules/cli/src/main/scala/scala/cli/internal/ProcUtil.scala

```diff
@@ -43,4 +48,39 @@ object ProcUtil {
     new String(data, StandardCharsets.UTF_8)
   }
 
+  def forceKillProcess(process: Process, logger: Logger): Unit = {
+    if (process.isAlive) {
+      process.destroyForcibly()
+      logger.debug("Force killing running process")
```

Comment:
NIT Maybe specify the PID in the debug message:
```suggestion
      logger.debug(s"Killing user process ${process.pid()}")
```